chain/actors/agen/generator/gen.go

```diff
@@ -83,7 +84,7 @@ func generateAdapters() error {
 				return err
 			}
 
-			if err := ioutil.WriteFile(filepath.Join(actDir, fmt.Sprintf("%s.go", act)), b.Bytes(), 0666); err != nil {
+			if err := ioutil.WriteFile(filepath.Join(actDir, fmt.Sprintf("%s.go", act)), b.Bytes(), 0o666); err != nil {
```

Comment:
curious what the difference is here?

Reply:
It's a consistent style thing from gofumpt or gofmt.